msgid ""
msgstr ""
"Project-Id-Version: red-discordbot\n"
"POT-Creation-Date: 2021-05-18 22:56+0000\n"
"Last-Translator: \n"
"Language-Team: Russian\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: redgettext 3.3\n"
"Plural-Forms: nplurals=4; plural=((n%10==1 && n%100!=11) ? 0 : ((n%10 >= 2 && n%10 <=4 && (n%100 < 12 || n%100 > 14)) ? 1 : ((n%10 == 0 || (n%10 >= 5 && n%10 <=9)) || (n%100 >= 11 && n%100 <= 14)) ? 2 : 3));\n"
"X-Crowdin-Project: red-discordbot\n"
"X-Crowdin-Project-ID: 289505\n"
"X-Crowdin-Language: ru\n"
"X-Crowdin-File-ID: 698\n"
"Language: ru_RU\n"

#: redbot/cogs/audio/apis/interface.py:276
#: redbot/cogs/audio/apis/interface.py:606
msgid "Failing to get tracks, skipping remaining."
msgstr "Не удалось получить треки, пропускаю оставшиеся треки."

#: redbot/cogs/audio/apis/interface.py:309
#: redbot/cogs/audio/apis/spotify.py:175
msgid "The Spotify API key or client secret has not been set properly. \n"
"Use `{prefix}audioset spotifyapi` for instructions."
msgstr "API ключ Spotify или секрет клиента были установлены неправильно. \n"
"Для получения инструкций используйте `{prefix}audioset spotifyapi`."

#: redbot/cogs/audio/apis/interface.py:351
msgid "This doesn't seem to be a valid Spotify playlist/album URL or code."
msgstr "Это не похоже на действительный адрес или код Spotify."

#: redbot/cogs/audio/apis/interface.py:477
msgid "This doesn't seem to be a supported Spotify URL or code."
msgstr "Возможно, этот формат файла не поддерживается."

#: redbot/cogs/audio/apis/interface.py:567
msgid "The connection was reset while loading the playlist."
msgstr "Соединение было сброшено при загрузке плейлиста."

#: redbot/cogs/audio/apis/interface.py:578
msgid "Player timeout, skipping remaining tracks."
msgstr "Тайм-аут проигрывателя, пропуск оставшихся треков."

#: redbot/cogs/audio/apis/interface.py:672
msgid " {bad_tracks} tracks cannot be queued."
msgstr " {bad_tracks} невозможно добавить в очередь."

#: redbot/cogs/audio/apis/interface.py:680
msgid "Playlist Enqueued"
msgstr "Плейлист добавлен в очередь"

#: redbot/cogs/audio/apis/interface.py:681
msgid "Added {num} tracks to the queue.{maxlength_msg}"
msgstr "{num} треков добавлено в очередь.{maxlength_msg}"

#: redbot/cogs/audio/apis/interface.py:687
msgid "{time} until start of playlist playback: starts at #{position} in queue"
msgstr "{time} до начала воспроизведения плейлиста: начинается с #{position} в очереди"

#: redbot/cogs/audio/apis/interface.py:698
msgid "Nothing found.\n"
"The YouTube API key may be invalid or you may be rate limited on YouTube's search service.\n"
"Check the YouTube API key again and follow the instructions at `{prefix}audioset youtubeapi`."
msgstr "Ничего не найдено.\n"
"Ключ YouTube API может быть недействительным или вы можете оценить его в поисковой службе YouTube.\n"
"Проверьте YouTube API еще раз и следуйте инструкциям в `{prefix}audioset youtubeapi`."

#: redbot/cogs/audio/apis/youtube.py:62
msgid "Your YouTube Data API token is invalid.\n"
"Check the YouTube API key again and follow the instructions at `{prefix}audioset youtubeapi`."
msgstr "Ваш токен API YouTube Data недействителен.\n"
"Проверьте ключ YouTube API снова и следуйте инструкциям в `{prefix}audioset youtubeapi`."

#: redbot/cogs/audio/apis/youtube.py:74
msgid "YouTube API error code: 403\n"
"Your YouTube API key may have reached the account's query limit for today. Please check <https://developers.google.com/youtube/v3/getting-started#quota> for more information."
msgstr "Код ошибки YouTube API: 403\n"
"Ваш ключ YouTube API достиг лимита запросов на сегодня. Пожалуйста, перейдите по ссылке <https://developers.google.com/youtube/v3/getting-started#quota> для получения дополнительной информации."

